modules/packages/Sort.chpl

```diff
@@ -1216,9 +1218,9 @@ module ShellSort {
     // and see Marcin Ciura - Best Increments for the Average Case of Shellsort
     // for the choice of these increments.
     var n = 1 + end - start;
-    var js,hs:int;
+    var js,hs:idxType;
     var v,tmp:Data.eltType;
-    const incs = (701, 301, 132, 57, 23, 10, 4, 1);
+    const incs = (701, 301, 132, 57, 23, 10, 4, 1):(8*idxType);
```

Comment:
It looks like a problem for the below code that some of those incs come out negative for `idxType` of `int(8)`.

(I don't remember my shell sort to know if it's also a problem that they're not strictly descending for `uint(8)`.)

Reply:
This should be resolved now.